.test/config/config.yaml

```diff
@@ -31,6 +35,10 @@ diffexp:
       # Binary valued covariate that shall be used for fold change/effect size
       # based downstream analyses.
       primary_variable: condition
+  # significance level used for volcano, ma- and qq-plots
+  sig_level_volcano-plot: 0.05
```

Comment:
have a section `sig-level`, and below entries for qq, ma, and volcano plots.

Reply:
Good idea!